packages/core/api/calendarEvents.ts

```
@@ -0,0 +1,62 @@
+  CreateCalendarEventInput,
+  UpdateCalendarEventInput,
+} from '../types';
+
+// Define a base URL for the API. This might come from an environment variable in a real app.
+const BASE_URL = '/api'; // Adjust if your server is hosted elsewhere or has a different prefix
+
+const client = hc<AppType>(BASE_URL);
+
+// Function definitions
+
+export const getCalendarEvents = async (params?: CalendarEventQueryParams): Promise<CalendarEvent[]> => {
```

Biome (1.9.4)

[error] Use Array<T> syntax instead of shorthand T[] syntax.

Unsafe fix: Use Array<T> syntax to replace


(lint/style/useConsistentArrayType)